rosbag_snapshot/include/rosbag_snapshot/snapshotter.h

```diff
@@ -183,9 +186,11 @@ class ROSBAG_DECL MessageQueue
   void _clear();
   // Truncate front of queue as needed to fit a new message of specified size and time. Returns False if this is
   // impossible.
-  bool preparePush(int32_t size, ros::Time const& time);
+  bool preparePush(int32_t size, ros::Time const& time, std::string const& callerid);
   // Returns true if queue messages are latched, false if not latched or queue is empty. Does not obtain lock
   bool _is_latched();
+  // Removes all messages from the specified callerid
+  void _remove_callerid(std::string callerid);
```

Comment:
Should still be `const str::string& callerid`.

Reply:
There's a strange mix of `camelCase` methods here (which are correct per our [code standards](https://locusrobotics.atlassian.net/wiki/spaces/RST/pages/17001101/Code+Conventions#CodeConventions-CodeStyle)) and `snake_case` method with leading underscores. I see a bunch of the other methods have the trailing underscore convention; did we add those, or was that something that was already present?

Reply:
Right, I forgot that I should "const all the things!". Interesting that you wrote `const std::string& callerid` (which makes more sense to me and I think is preferred?) while the rest of the code does it like `std::string const& callerid`.

I only added the `_is_latched`, `pop_back`, and `_pop_back` methods before this PR and I was more or less following what I saw for the existing functions. I'll change them to camel case. However with the `pop_back` ones, I suppose I should keep the underscore for the private one, unless there's a better way to differentiate.